submissions/adeyemiJuwon/javascript/productlist.js

```diff
@@ -0,0 +1,24 @@
+
+const productList = [
+  {
+    productName: "Xiaomi Redmibook",
+    imageUrl: "img/laptop1.jpg",
+    productPrice: "£819.52 GBP"
+    
+  },
+  {
+    productName: "1.8m DC Charging power",
+    imageUrl: "img/charger.jpg",
+    productPrice: "£4.26 GBP"
+  },
+  {
+    productName: "11 Inch Laptop Bag Tablet Zipper Pouch Sleeve for MacBook Air",
+    imageUrl: "img/bag1.jpg",
+    productPrice: "£5.95"
+  }
+];
+
+const newList = productList.forEach((product) => {
```

Comment:
There is no need for variable declaration `const newList` cos it is not been used. Also, format your code(line 22)